Makefile

```diff
@@ -28,9 +28,13 @@ export KUBEVIRT_PROVIDER ?= k8s-1.19
 export KUBEVIRT_NUM_NODES ?= 2 # 1 master, 1 worker needed for e2e tests
 export KUBEVIRT_NUM_SECONDARY_NICS ?= 2
 
-export E2E_TEST_TIMEOUT ?= 40m
+export E2E_TEST_TIMEOUT ?= 60m
 
+ifeq ($(findstring true,$(NMSTATE_PARALLEL_ROLLOUT)),true)
```

Comment:
Let's do the skip just in automation, so it's only skipped at CI.

So by default a user will run all the test in sequential rollout and the small parallel test that check enactments.

But thinking about it what we really should do is to trait parallel and sequential rollout at different tests with DescriptionTable or the like, so users just run all of them and they are different test with different names and results, but that's a lot of work.

Reply:
> Let's do the skip just in automation, so it's only skipped at CI.

Done

> But thinking about it what we really should do is to trait parallel and sequential rollout at different tests with  DescriptionTable or the like, so users just run all of them and they are different test with different names and results, but that's a lot of work.

I agree, I'll add this to the list of followus in the PR description